server/server.go

```diff
@@ -218,6 +219,12 @@ func (svr *Server) HandlePut(w http.ResponseWriter, r *http.Request) (commitment
 	commitment, err := svr.router.Put(r.Context(), meta.Mode, comm, input)
 	if err != nil {
 		err = fmt.Errorf("put request failed with commitment %v (commitment mode %v): %w", comm, meta.Mode, err)
+
+		if utils.ErrorMessageContainsAny(err, store.OversizedEigenDAError, store.OversizedProxyError) {
```

Comment:
```suggestion
                // we add here any error that should be returned as a 400 instead of a 500.
                // currently only includes oversized requests
		if utils.ErrorMessageContainsAny(err, store.OversizedEigenDAError, store.OversizedProxyError) {
```

Reply:
[97c51a0](https://github.com/Layr-Labs/eigenda-proxy/pull/140/commits/97c51a0994e1a02077ce6f6a432a1ff6e44d8fd4)